src/pages/channels/channelPSPList/ChannelPSPTable.tsx

```diff
@@ -157,7 +157,7 @@ export default function ChannelPSPTable({ setAlertMessage, pspNameFilter }: Chan
                   pspListPage.page_info?.total_pages && pspListPage.page_info?.total_pages > 1 ? (
                     <Pagination
                       color="primary"
-                      count={pspListPage.page_info?.total_pages ?? 0}
+                      count={pspListPage?.page_info?.total_pages ?? 1}
```

Comment:
why `1`?

Reply:
If the totalPages is undefined/null the default value should be 1 for the pages, with 0 the pagination is broken and doesn't show any pages (even if there are elements for 1 page)
Just a precaution